ecvi2.xsd

```diff
@@ -1,1646 +1,1753 @@
+            07/09/2020: Added ReplacesCviNumber attribute. #52 
+            07/09/2020: Added Voided attribute. #51 
+            07/09/2020: Added Product and supporting CommodityType enumeration. #50 
+            07/09/2020: Added TestName attribute. #44
+            08/13/2020: Added choice element requiring one or more Animal, GroupLot or Product. #54
+            08/13/2020: Updated annotations in several places. #51, #52, #53 
+            08/13/2020: Added TransportModeOtherDescription
+            01/02/2021: Documentation element updates #8, #61
+            3/17/2021: Converted Veterinarian to Natural Person. First and last names required. #63 
+            5/13/2021: Relax Address element for all but Origin, Destination and Field (accession) #69 
+            8/31/2021: Added optional BusinessName to Veterinarian. #63 
+            8/18/2022: Added all state abbreviations to NUES9 regular expression. #74
+        </xs:documentation>
+    </xs:annotation>
+    <xs:annotation>
+        <xs:documentation>Copyright © 2017-22 AAVLD/USAHA it is free for use. Submit suggested
+            changes to the maintainers at https://github.com/AAVLD-USAHA-ITStandards/eCVI
+        </xs:documentation>
+    </xs:annotation>
+
+    <xs:element name="eCVI">
+        <xs:annotation>
+            <xs:documentation> The eCVI xml document contains an extract of data from an interstate
+                certificate of veterinary inspection (normally electronic) used to communicate the
+                information to another computer system. It does not constitute the legal document;
+                that remains the job of the originating application. This schema tries to capture
+                those data elements and the level of detail needed for regulatory decision support,
+                disease traceability, and business interoperability. A large majority of use-cases
+                can be met using the fully-structured (close standard) data. Inevitably, edge
+                cases--even very important ones--have had to be left to open standard features such
+                as "other" choices and free-text entry elements.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="Veterinarian" minOccurs="1" maxOccurs="1"/>
+                <xs:element ref="MovementPurposes" minOccurs="1" maxOccurs="1"/>
+                <xs:element ref="Origin" minOccurs="1" maxOccurs="1"/>
+                <xs:element ref="Destination" minOccurs="1" maxOccurs="1"/>
+                <xs:element ref="Consignor" minOccurs="0" maxOccurs="1"/>
+                <xs:element ref="Consignee" minOccurs="0" maxOccurs="1"/>
+                <xs:element ref="Carrier" minOccurs="0" maxOccurs="1"/>
+                <xs:element ref="TransportMode" minOccurs="0" maxOccurs="1"/>
+                <xs:element ref="TransportModeOtherDescription" minOccurs="0" maxOccurs="0"/>

```

Comment:
Was implementing this in preparation for it becoming active Jan 1 and realized that the `maxOccurs` is set to 0 for `TransportModeOtherDescription`, so the xml document will never validate if this element is included.
```suggestion
                <xs:element ref="TransportModeOtherDescription" minOccurs="0" maxOccurs="1"/>

```